### ISSUE TITLE
what threshold for the score to accept the prediction ? 

### ISSUE THREAD

**Issue by Hadjer13** [facebookresearch/BLINK#37]
I'm using BLINK for entity mapping
I want to know what is the interval of values for the _prediction scores_, to be able to fix an acceptable threshold to consider the predicted linking as true. When looking at some example, I've noticed that the values could be positive or negative, and when the returned label from Wikipedia is true, the score is highly positive, is my assumption true ? how we interpret the values of the score ? 
Thanks in advance

**Comment by ledw** [comment]
@Hadjer13 Are you using fast mode or not? If not, the prediction scores should be the logit values of the top 10 predictions, so you can apply a softmax on top of it to get probabilities. 
_"when the returned label from Wikipedia is true"_ : What is this referring to?
